index.html

```diff
@@ -34,7 +34,7 @@
 </p>
 
 {% highlight bash %}
-zypper install yast2-devtools rubygem-yast-rake git-core
+zypper install yast2-devtools ruby2.1-rubygem-yast-rake git-core
```

Comment:
Please, verify that my suggestion works and update the pull request according to comments. Maybe adding a note for the reader to beware the spaces: there should be NO spaces before or after `=` (something most Linux users should now, but...)